### ISSUE TITLE
Github Pages not getting updated correctly

### ISSUE THREAD

**Issue by dbluhm** [decentralized-identity/peer-did-method-spec#65]
As pointed out by @FabioPinheiro in https://github.com/decentralized-identity/peer-did-method-spec/issues/64#issuecomment-1782963289, a recent merged PR updating the did:peer:2 spec should have updated the github pages. However, the built artifact from the action still has the old spec contents.

**Comment by swcurran** [comment]
Pretty sure it’s because the main branch is still `master` in the repo, and the action triggers on `main`. Changing the name of the branch to see if that fixes it.  Might take a tweek PR to trigger the build.

**Comment by swcurran** [comment]
Resolved.

- The GitHub Action was triggered based on a merge into `main`, so I changed that to `master`. I don’t have the power to change the branch name to `main`, but we should do that — and change the GHA to trigger on `main`.
- Once executed, the GHA was pushing the file to the `gh-pages` branch (yay!), but GH Pages settings published from the `master` branch.  I updated the source of the Pages to be the `gh-pages` branch.

Looks good to me know, so closing this.
